src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests/Services/ResetSearchableDocuments/FullSearchableDocumentResetterTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Domain;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Services.ResetSearchableDocuments;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests.Builders;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests.Services.ResetSearchableDocuments;
+
+public class FullSearchableDocumentResetterTests
+{
+    private readonly ContentApiClientMockBuilder _contentApi = new();
+    private readonly SearchableDocumentRemoverMockBuilder _searchableDocumentRemover = new();
+
+    private IFullSearchableDocumentResetter GetSut() =>
+        new FullSearchableDocumentResetter(
+            _contentApi.Build(),
+            _searchableDocumentRemover.Build(),
+            new NullLogger<FullSearchableDocumentResetter>());
+
+    [Fact]
+    public void Can_instantiate_SUT() => Assert.NotNull(GetSut());
+
+    [Fact]
+    public async Task WhenPerformFullReset_ThenAllSearchableDocumentsAreRemoved()
```

Comment:
This test method and the one below could be named `WhenPerformReset_`... because the method name  is `PerformReset`.